online-banking-cz/src/main/res/layout/online_banking_cz_spinner_layout.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2022 Adyen N.V.
+  ~
+  ~ This file is open source and available under the MIT license. See the LICENSE file for more info.
+  ~
+  ~ Created by atef on 8/9/2022.
+  -->
+
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    tools:parentTag="android.widget.LinearLayout">
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/textInputLayout_issuers"
+        style="@style/AdyenCheckout.DropdownTextInputLayout.Issuer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <androidx.appcompat.widget.AppCompatAutoCompleteTextView
+            android:id="@+id/autoCompleteTextView_issuers"
+            style="@style/AdyenCheckout.DropdownTextInputEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:dropDownAnchor="@id/textInputLayout_issuers" />
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <TextView
+        android:id="@+id/textview_termsAndConditions"
+        style="@style/AdyenCheckout.Issuer.TermsAndConditions"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        tools:text="A.Etman" />
```

Comment:
It's more useful to have the actual text here, so the preview looks more like how it would like when you run the app